### PR TITLE
Configure benchmarks module and CI

### PR DESCRIPTION
**Pull request by guizmaii**
See:
- https://github.com/fwbrasil/kyo
- https://github.com/marketplace/actions/continuous-benchmark#charts-on-github-pages-1
- https://github.com/sbt/sbt-jmh

### CODE REVIEW

File: .github/workflows/benchs.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        with:
+          distribution: temurin
+          java-version: 17
+          check-latest: true
+
+      - name: compile
+        run: sbt compile
+
+      - name: run benchmarks
+        run: sbt zioKafkaBench/"jmh:run -wi 5 -i 3 -r 1 -w 1 -t 1 -rf json -foe true" # TODO Jules: TO REFINE?
```
Comment (guizmaii) [inline .github/workflows/benchs.yml:34]: TODO @guizmaii

File: .github/workflows/benchs.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          output-file-path: zioKafkaBench/jmh-result.json
+          # Access token to deploy GitHub Pages branch
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # Push and deploy GitHub pages branch automatically
+          auto-push: true
+          alert-threshold: '200%'
+          comment-on-alert: true
+          fail-on-alert: true
+          # Mention these maintainers in the commit comment
+          alert-comment-cc-users: '@iravid,@svroonland,@guizmaii'
```
Comment (guizmaii) [inline .github/workflows/benchs.yml:66]: @svroonland @iravid Are you ok being in this list? You'll be mentioned in alert that this Github Actions will create when perf issues are detected. See example here https://github.com/fwbrasil/kyo/commit/54b42e43b3b0596de5956124268b93a7f1477837#commitcomment-98461052

Should I include anyone else? @jdegoes @adamgfraser Are you interested in being in this list? Do you see anyone I should add?

Comment (svroonland) [reply .github/workflows/benchs.yml:66]: Fine for me!

**Review by svroonland — APPROVED** [review]
Nice!

**Pull request merged**